src/workerd/io/actor-cache.c++

```diff
@@ -2492,6 +2499,11 @@ kj::Promise<void> ActorCache::startFlushTransaction() {
 }
 
 kj::Promise<void> ActorCache::flushImpl(uint retryCount) {
+  auto durationAlert = util::DurationExceededLogger(
+    clock,
+    5 * kj::SECONDS,
+    "flushImpl took longer than expected"
```

Comment:
This would be much more useful if it included some summary of the operations included in the flush. Or if it allowed us to distinguish between the actual flushing being slow and the `waitForPastReads` step being slow.